src/Sulu/Bundle/TrashBundle/Resources/js/index.js

```diff
@@ -6,5 +6,5 @@ import RestoreItemAction from './views/List/itemActions/RestoreItemAction';
 listItemActionRegistry.add('sulu_trash.restore', RestoreItemAction);
 
 initializer.addUpdateConfigHook('sulu_trash', (config: Object) => {
-    RestoreItemAction.restoreFormMapping = config.restoreFormMapping;
+    RestoreItemAction.restoreConfigurationMapping = config.restoreConfigurationMapping;
```

Comment:
this should be adjusted to prevent the following issue: https://github.com/sulu/sulu/issues/6275

Reply:
yes, it should be quite easy to fix this. just added this comment as reminder 🙂